### ISSUE TITLE
Improve accessibility of genesis-oik

### ISSUE THREAD

**Issue by bobbingwide** [bobbingwide/genesis-oik#21]
Using WAVE and axe DevTools evaluate the accessibility of sites that use genesis-oik and improve the theme to reduce the errors.  

Local dev: s.b/oikcom
Clones: oik-plugins.co.uk and oik-plugins.com

**Comment by bobbingwide** [comment]
- Background colour of buttons ( a.button ) is too light `#baba54`. 
- It should be `#787830` for AA or `#5b5b25` for AAA
- We'll need to change the hover color to a darker value too. 
- Currently `#5b6021`
- I'll try `#353813`.

**Comment by bobbingwide** [comment]
- Also need to change `#e5554e`, which is used in footer widgets and on the nav menu when hovered to a darker value.
- `#df332a` for AA or `#a92019` for AAA

**Comment by bobbingwide** [comment]
Change background of downloads from wordpress.org in `oik-custom.css`
- currently `#eb672b`
- `#c94c13` for AA, `#95390e` for AAA

**Comment by bobbingwide** [comment]
Having resolved most of the Errors and Color Contrast errors I started to work on the Alerts.
 - On the home page, which doesn't have a h1 in the body, I was getting a _Possible heading_ alert.
 - In fact, I was getting this on every page.  
 - The site-title was marked up with a `<p>` tag.
 - So I set about seeing how to set the tag for `class="site-title"` to `<h1>`.
 - The logic in `genesis_seo_site_title()` didn't appear to be working.
 - Eventually, after a lot of trial and error I discovered that the `genesis-seo-settings` option field was an empty array.
 - This was odd, since the Customizer displayed the value I expected

<img width="369" height="357" alt="Image" src="https://github.com/user-attachments/assets/c9bbe628-6b4f-41be-8781-935a6415efa5" />

- It would appear that the Customizer is showing the default value.
- And I hadn't chosen Publish
- Once done ( I set it to Description first as a test ) I was able to get the result I wanted on the home page.

**Note:** I'd disabled WordPress SEO in my local development environment to avoid a Color Contrast error from its traffic light in the dashboard top menu bar. Had I not done this I might not have had to spend so much time tracing `genesis_seo_site_title()`!

**Comment by bobbingwide** [comment]
Regarding WAVE Alerts. 

One most pages there are several _Redundant link_ alerts.
- When I'm logged in I get two (Edit) links at the bottom of each post.
- I never managed to get rid of the first one - produced by genesis
- Until today that is! Hurrah

Other alerts on the home page are:
- 1 Skipped heading level - a h3 which should be h2
- 2 Possible heading - produced by [bw_slogan] and [bw_alt_slogan]
- 4 Suspicious link text - potentially unnecessary `read more` links. 
- 21 Redundant link

**Comment by bobbingwide** [comment]
One of the Redundant link alerts, on the home page and every other page, is for the Home link in the primary menu.

<img width="398" height="144" alt="Image" src="https://github.com/user-attachments/assets/9b487924-a4f7-4fd1-9973-3438f5e9020b" />

ChatGPT suggested several ways of dealing with this.

1. Leave both in place... and live with the Alert
2. Hide the site-title link from assistive tech
3. Hide the home item from assistive tech

I'll try 2. 

```
add_filter( 'genesis_seo_title_atts', function( $atts ) {
    // Hide the header-site-title link from screen readers
    $atts['aria-hidden'] = 'true';
    $atts['tabindex']    = '-1';
    return $atts;
});
```

**Comment by bobbingwide** [comment]
> I'll try 2.

Several things wrong with this.

1.  ChatGPT got the code wrong. There is no `genesis_seo_title_atts` filter.
2. I added `aria-hidden="true" tabindex="-1"` to the link in the site title; WAVE still reported the Home link in the menu as a Redundant link
3. I manually attempted to use `aria-hidden` on the Home link; WAVE still reported it as a duplicate.

I may as well remove the Home item from the menu.
